dynamic/coinchange.go

```diff
@@ -1,15 +1,12 @@
 package dynamic
 
-// return combination of coin
+// CoinChange 
```

Comment:
I think you are not getting the idea of go doc comments. A few examples may help:
https://github.com/TheAlgorithms/Go/blob/master/math/prime/millerrabinprimalitytest.go#L99
https://github.com/TheAlgorithms/Go/blob/master/math/prime/millerrabinprimalitytest.go#L119

Reply:
Can you give me suggestion, what should i write. i still can't understand

is this correct?
```
// CoinChange find number of combination coins to get amount target
```

Reply:
I think the language can be improved upon.

Reply:
i only can think to write like this, make a simple comment, i look another example they make comment like this too just simple and describe what does the function do
```
// CoinChange find number of combination coins to get amount target
```

Reply:
```go
// CoinChange finds the number of possible combinations of coins
// of different values which can get to the target amount.
```